@here/harp-examples/src/geo-to-screen.ts

```diff
@@ -0,0 +1,286 @@
+/*
```

Comment:
I think we need to discuss what examples we want to have and how we want to structure them.
Also the examples should tell a story à la scrum: `As a user I want to add a marker in screen space at this geo location` 

At least we should have one section in the examples about the whole markers/map-objects topic.

Reply:
yes, definitely, we need some structure, I like writing these kind of little test apps to visually verify the changes, but ofter there is not yet a good place to put them, and I think, in the internal example repository things bitrot